tsconfig.json

```diff
@@ -1,6 +1,6 @@
 {
   "compilerOptions": {
-    "target": "es6",
+    "target": "esnext",
```

Comment:
Do we still need this change?

Reply:
yep, we fail to compile without it

Reply:
Changed `tsconfig.json` target back to `es6` and updated `tsconfig.eslint.json` to target `esnext` so it doesn't complain about `AsyncIterable` types.